README.md

```diff
@@ -393,41 +393,156 @@ Expected Response:
 </details><br>
 
 <details close>
-  <summary>Remove Game and User Association</summary><br>
+  <summary>Delete a User</summary><br>
 
-Delete a game from a user's owned games list.<br>
+Deletes a user in the database.<br>
 
 ```query
 mutation {
-          deleteUserGame(input :{
-              id: 1,
-              }) {
-              id
-          }
-      }
+  deleteUser(input: {id: 2}) {
+      id
+      errors
+   }
+}
 ```
 
 Expected Response:
 
  ```json
 {
-    "data": {
-        "deleteUserGame": {
-            "id": "1"
-        }
+  "data": {
+    "deleteUser": {
+      "id": "3",
```

Comment:
Should this match the ID on line 402?

Reply:
Yes, thank you for catching that! It has been updated to match